--- a/Editor/CMakeLists.txt
+++ b/Editor/CMakeLists.txt
@@ -99,8 +99,7 @@ endif()
 
 if(UNIX)
     set(CMAKE_SKIP_INSTALL_ALL_DEPENDENCY true)
-    set(CMAKE_INSTALL_PREFIX /usr/bin)
-    install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX} COMPONENT binaries)
+    install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin COMPONENT binaries)
 endif()
 
 if(APPLE)
